src/qute/commands/registerCommands.ts

```diff
@@ -91,17 +165,65 @@ export function registerConfigurationUpdateCommand(context: ExtensionContext) {
         config.workspaceConfiguration.update(section, value, config.target);
         break;
       }
+      case ConfigurationItemEditType.Remove: {
+        removeFromPreferenceArray(config, section, value);
+        break;
+      }
     }
   }));
 }
 
-function getSettingsValue(value: any, section: string) {
+/**
+   * Sets the `editorLangIdSupportsQute` context to `true` if the editor language id
+   * is a qute template. Sets to `false` otherwise.
+   */
+export async function updateQuteContext(document: TextDocument) {
```

Comment:
What's the difference between this and `synchronizeQuteValidationButton` ? With the new change this isn't used anywhere. Can it be removed ?

Reply:
The merge from my POC was not done correctly,now we have synchronizeQuteValidationButton instead of updateQuteContext.

Reply:
updateQuteContext must be removed